test/core/modules/webgl-distort.js

```diff
@@ -0,0 +1,49 @@
+var test = require('tape');
+var base64Img = require('base64-img');
+var looksSame = require('looks-same');
+
+require('../../../src/ImageSequencer.js');
+
+var sequencer = ImageSequencer({ ui: false });
+var options = {
+        nw: '0 100',
+        ne: '1023 50',
+        se: '1223 867',
+        sw: '100 767'
+    };
+var target = 'test_outputs';
+var red = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQAQMAAAAlPW0iAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABlBMVEX+AAD///+KQee0AAAAAWJLR0QB/wIt3gAAAAd0SU1FB+EGHRIVAvrm6EMAAAAMSURBVAjXY2AgDQAAADAAAceqhY4AAAAldEVYdGRhdGU6Y3JlYXRlADIwMTctMDYtMjlUMTg6MjE6MDIrMDI6MDDGD83DAAAAJXRFWHRkYXRlOm1vZGlmeQAyMDE3LTA2LTI5VDE4OjIxOjAyKzAyOjAwt1J1fwAAAABJRU5ErkJggg==";
+var benchmark = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAH0lEQVQ4T2NkoBAwUqifYdQAhtEwYBgNA1A+Gvi8AAAmmAARf9qcXAAAAABJRU5ErkJggg==";
+
+// Test 1 to check webgl-distort module is getting loaded
+test('Load distort module', function(t) {
+  sequencer.loadImages(red);
+  sequencer.addSteps('webgl-distort', options);
+  t.equal(sequencer.steps[1].options.name, 'webgl-distort', 'Distort module is getting loaded');
+  t.end();
+});
+
+// Test 2 to check options are correct
+test('Check Options', function(t) {
+  t.equal(sequencer.steps[1].options.nw, '0 100', 'Options are correct');
+  t.equal(sequencer.steps[1].options.ne, '1023 50', 'Options are correct');
```

Comment:
Here, that's a really big distortion; it's a very tiny image; could we just say something under 100? 

Also, would it be possible to pass comma-separated values rather than space-separated? I think it'll render better in a URL, don't you think?

Reply:
Agreed, comma separated it is, then

Reply:
Comma separated values can have spaces as well. We have to replace those as well using regex. A better approach would be to do both. We can first separate it at spaces, then remove all commas, if there are no spaces, separate at commas.